examples/ios/BUILD.bazel

```diff
@@ -0,0 +1,92 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application")
+load("@build_bazel_rules_apple//apple:macos.bzl", "macos_application")
+load("@rules_cc//cc:defs.bzl", "cc_library", "objc_library")
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+cc_library(
+    name = "allocator_library",
+    srcs = ["allocator_library.cc"],
+    tags = ["manual"],
+)
+
+rust_library(
+    name = "rust_lib",
+    srcs = ["demo.rs"],
+    edition = "2018",
+    tags = ["manual"],
+    deps = [":allocator_library"],
+)
+
+# This being required is a bug in objc_library, ideally 'main_lib' could depend directly on 'rust_lib'
```

Comment:
Do you know if this is tracked by an upstream issue in Bazel? Or should one be filed for `rules_rust`?

Reply:
https://github.com/bazelbuild/bazel/pull/15462

Reply:
Can we add that the code here? Something like the following maybe?:
```suggestion
# TODO(bazelbuild/bazel#15462): This being required is a bug in objc_library, ideally 'main_lib' could depend directly on 'rust_lib'
```

Reply:
Adding a TODO to remove, I verified it works without this shim with the latest rolling release